wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -122,15 +122,6 @@ private void signInOnline(Bundle parameters) throws Exception {
         }
     }
 
-    private void confirmLocalFileAccess() throws Exception {
-        // First time run requires confirmation to allow access to local files
-        UiObject allowButton = new UiObject(new UiSelector().textContains("Allow")
-                                                            .className("android.widget.Button"));
-        if (allowButton.waitForExists(timeout)) {
```

Comment:
If condtion vs while in the base class method. I don't think this should cause any problems?